### DISCUSSION TITLE
How to remove Valetudo

### DISCUSSION THREAD

**Discussion by Ulrar** [Hypfer/Valetudo#1779] [Q&A / Support] [answered]
Hi,

I installed Valetudo in my mom's Dreame D9 and she loves it, however the other day a lot of water came out of the mop compartment and got sucked in, and it since refuses to run saying to check the filter. The Valetudo UI says the fan speed is abnormal.
She cleaned the filter and dried everything but no change, so sadly it seems it'll have to go back under warranty.

Tl;dr I'm guessing they wouldn't take too kindly to finding Valetudo on there, so what's the correct way to go back to stock firmware ?
Had a look through the website and I don't see any obvious answers (apologies if I missed it), although I understand updating the firmware would do it. Would following the reset procedure from the manual be enough?

Thank you !

**Comment by Hypfer** [comment, marked as answer]
Please open a ticket at https://vaers.dontvacuum.me/

**Comment by 13peter13** [comment]
Hello Ulrar,

I also hopefully have a warranty case and have to send it back. How did you proceed to remove valetudo?

Thanks in advance!

**Reply by Ulrar** [reply]
Yeah you have to open a ticket over there, I got a clean build generated for me.
Haven't used it yet as I'm in another country, but will this week

**Comment by Dennis182** [comment]
Hi,

is this also the way to go for the L10 Pro?

Thank you in advance.
